Homeworks/Homework SQL/SQLQuery-Homework.sql

```diff
@@ -0,0 +1,339 @@
+ALTER TABLE Books
+DROP COLUMN AuthorFirstName;
+ALTER TABLE Books
+DROP COLUMN AuthorLastName;
+ALTER TABLE Books
+DROP COLUMN Type;
+
+CREATE TABLE Author
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+FirstName VARCHAR (50),
+LastName VARCHAR (50),
+Email VARCHAR (30),
+)
+
+CREATE TABLE TYPE
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR (30),
+)
+
+ALTER TABLE Books
+ADD AuthorId INT REFERENCES Author(id);
+
+ALTER TABLE Books
+ADD TypeId INT REFERENCES Type(id);
+
+SELECT *
+FROM Books;
+
+ALTER TABLE Librarians
+DROP COLUMN IsOnHoliday;
+
+CREATE TABLE IsOnHoliday
```

Comment:
It is enough to look at the table name and you'll realize that something is wrong with this table. We usually keep entities / resources in a table (like books, animals, cars. authors, restaurants, companies, employees etc.) and this "IsOnHoliday" is clearly not a resource. I can see you have a HolidayStatus as a column so maybe HolidayStatuses can be used as a better name. 
This table is not useful in my opinion but if you want to keep it make sure that you change its name.